converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java

```diff
@@ -119,7 +119,7 @@ private void encodeReportingParameter(JsonWrapper wrapper, Node node) {
 			DEV_LOG.error("Missing Reporting Parameters from ACI Section");
 			return;
 		}
-		reportingParamEncoder.encode(wrapper, reportingChild);
+		reportingParamEncoder.encode(wrapper, reportingChild, false);
```

Comment:
Do you think we run into the same problem for the IA and Quality sections too?

Reply:
Yes, those sections would also be vulnerable

Reply:
Should we address them in this PR or a different PR?  I think these changes would be easy to extend to the other sections.

Reply:
They both extend from AciSectionEncoder

Reply:
Oh......

![Not very bright](https://media.giphy.com/media/Zv6Apawffpa2k/giphy.gif)